ROSCO_toolbox/utilities.py

```diff
@@ -207,8 +206,12 @@ def write_DISCON(turbine, controller, param_file='DISCON.IN', txt_filename='Cp_C
     file.write('"{}"            ! DLL_FileName        - Name/location of the dynamic library in the Bladed-DLL format\n'.format(rosco_vt['DLL_FileName']))
     file.write('"{}"            ! DLL_InFile          - Name of input file sent to the DLL (-)\n'.format(rosco_vt['DLL_InFile']))
     file.write('"{}"            ! DLL_ProcName        - Name of procedure in DLL to be called (-) \n'.format(rosco_vt['DLL_ProcName']))    
+    file.write('\n')
+    file.write('!------- ZeroMQ Interface ---------------------------------------------------------\n')
+    file.write('"{}"            ! ZMQ_CommAddress     - Communication address for ZMQ server \n'.format(rosco_vt['ZMQ_CommAddress']))
```

Comment:
What format is this supposed to be? Is this a port address?  An example would be pretty helpful.

Reply:
Added an example in the variable description